docs/notifications.json

```diff
@@ -18,5 +18,10 @@
     "id": 76,
     "title": "<b>Unveiling Charts: Alpha release is live</b>",
     "text": "We're starting with bars, lines, and scatter charts. <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-introduce-charts\" href=\"https://mui.com/x/react-charts/\">Try X Charts now</a>, and let us know what you need."
+  },
+  {
+    "id": 77,
+    "title": "<b>MUI Toolpad is now in Beta</b>",
+    "text": "Import in your codebase, connect to datasources and build UI in minutes! Try <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-toolpad-introduce-beta\" href=\"https://mui.com/toolpad/\"> Toolpad today</a>, and share your feedback."
```

Comment:
```suggestion
    "text": "Import in your codebase, connect to datasources and build UI in minutes! Try <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-toolpad-introduce-beta\" href=\"https://mui.com/toolpad/\">Toolpad</a> today, and share your feedback."
```

Reply:
Didn't mean to remove the blog link. This suggestion was made before

Reply:
Yes yes :) I think taking users to the landing page should be better.